[2023-12-10 19:40:04,146] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-03-27T00:00:00+00:00 [queued]>
[2023-12-10 19:40:04,675] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-03-27T00:00:00+00:00 [queued]>
[2023-12-10 19:40:04,712] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:40:04,737] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-10 19:40:04,746] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:40:05,104] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2023-03-27 00:00:00+00:00
[2023-12-10 19:40:05,121] {standard_task_runner.py:52} INFO - Started process 2620 to run task
[2023-12-10 19:40:05,181] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2023-03-27T00:00:00+00:00', '--job-id', '721', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpeg7hlq3t', '--error-file', '/tmp/tmp82hf1twa']
[2023-12-10 19:40:05,191] {standard_task_runner.py:80} INFO - Job 721: Subtask segunda_tarea
[2023-12-10 19:40:06,420] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-03-27T00:00:00+00:00 [running]> on host 42828894a739
[2023-12-10 19:40:09,032] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2023-03-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-27T00:00:00+00:00
[2023-12-10 19:40:09,065] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-12-10 19:40:09,263] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-12-10 19:40:09,626] {subprocess.py:85} INFO - Output:
[2023-12-10 19:40:09,849] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-12-10 19:40:09,865] {subprocess.py:96} INFO - Command exited with return code 0
[2023-12-10 19:40:10,701] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20230327T000000, start_date=20231210T194004, end_date=20231210T194010
[2023-12-10 19:40:10,935] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-10 19:40:11,759] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
